__tests__/app.test.js

```diff
@@ -2,8 +2,14 @@ const request = require("supertest");
 const app = require("../app");
 const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data/index");
+const db = require("../db/connection");
 
-seed(testData);
+beforeAll(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  db.end();
```

Comment:
this won't end the connection without a return as it is on a new line, same as the function above.